src/main/java/com/puppycrawl/tools/checkstyle/utils/CodePointUtil.java

```diff
@@ -55,4 +55,32 @@ public static boolean hasWhitespaceBefore(int index, int... codePoints) {
                 .allMatch(Character::isWhitespace);
     }
 
+    /**
+     * Removes trailing whitespaces.
+     *
+     * @param codePoints array of unicode code points
+     * @return unicode code points array with trailing whitespaces removed
+     */
+    public static int[] stripTrailing(int... codePoints) {
+        int lastIndex = codePoints.length;
+        while (CommonUtil.isCodePointWhitespace(codePoints, lastIndex - 1)) {
+            lastIndex -= 1;
```

Comment:
@MUzairS15 why not postifx (`lastIndex--`)?

Reply:
@nick-mancuso I don't remember when, but once when I used postfix operator,  I got a violation, I'll build at local and do the changes if build passes

Reply:
done

Reply:
Resolved.